src/test/integration/tests/Deposit_Delegate_Allocate.t.sol

```diff
@@ -4,21 +4,24 @@ pragma solidity ^0.8.27;
 import "src/test/integration/IntegrationChecks.t.sol";
 import "src/test/integration/users/User.t.sol";
 
-contract Integration_Deposit_Delegate_Allocate is IntegrationCheckUtils {
-    function testFuzz_deposit_delegate_allocate(uint24 _random) public {
+// TODO: move randomness from tests
+
+contract Integration_Deposit_Delegate_Allocate is IntegrationCheckUtils, IDelegationManagerTypes {
```

Comment:
Same thing here and the other test file, could we remove the IDelegationManagerTypes inheritance?

Reply:
fixed in [d7a5d61](https://github.com/Layr-Labs/eigenlayer-contracts/pull/934/commits/d7a5d61437737a470dd4d22220e8a818d21c2ac1)